amespahdbpythonsuite/amespahdb.py

```diff
@@ -0,0 +1,707 @@
+import os
+import copy
+import hashlib
+import tempfile
+import time
+from datetime import timedelta
+import re
+
+import pickle
+
+from amespahdbpythonsuite.xmlparser import XMLparser
+
+
+class AmesPAHdb:
+    """
+    AmesPAHdbPythonSuite main class.
+    Contains methods to parse the database, perform search based on query,
+    retrieve UIDs, and call :meth:`amespahdbpythonsuite.transitions`
```

Comment:
:meth:`amespahdbpythonsuite.transitions` is not the only method that can be called.